src/higley_lab_to_nwb/benisty_2022/benisty_2022_spike2ttlinterface.py

```diff
@@ -0,0 +1,79 @@
+from typing import List
+import numpy as np
+from neo import io
+
+from neuroconv.datainterfaces.ecephys.baserecordingextractorinterface import BaseRecordingExtractorInterface
+from neuroconv.tools import get_package
+from neuroconv.utils import FilePathType
+from neuroconv.tools.signal_processing import get_rising_frames_from_ttl
+from spikeinterface.extractors import CedRecordingExtractor
+def _test_sonpy_installation() -> None:
+    get_package(
+        package_name="sonpy",
+        excluded_python_versions=["3.10", "3.11"],
+        excluded_platforms_and_python_versions=dict(darwin=dict(arm=["3.8", "3.9", "3.10", "3.11"])),
+    )
+def get_stream_ids(file_path: FilePathType) -> List[str]:
+    """Return a list of channel names as set in the recording extractor."""
+    r = io.CedIO(filename=file_path)
+    signal_streams = r.header["signal_streams"]
+    stream_ids = list(signal_streams["id"])
+    return stream_ids
+
+def get_stream_names(file_path: FilePathType) -> List[str]:
+    """Return a list of channel ids as set in the recording extractor."""
+    r = io.CedIO(filename=file_path)
+    signal_channels = r.header["signal_channels"]
+    channel_names = signal_channels["name"]
+    return channel_names
+
+class Benisty2022Spike2TTLInterface(BaseRecordingExtractorInterface):
```

Comment:
I'm also missing the `get_metadata()` function where you could add a description for the Wheel signal.

Reply:
Ah yes, right!